requirements.txt

```diff
@@ -1,8 +1,8 @@
 --editable git+https://github.com/openfisca/openfisca-core.git@master#egg=OpenFisca-Core[dev]
 docutils==0.12
-guzzle_sphinx_theme
-recommonmark
-sphinx-autobuild
-sphinx-markdown-tables
+guzzle_sphinx_theme==0.7.11
+recommonmark==0.4
```

Comment:
This is what was causing recent builds to fail. recommonmark 0.5 breaks some links.